REPORTING.md

```diff
@@ -0,0 +1,39 @@
+# How to generate Snyk and Accessibility reports
+
+## Snyk:
+
+This script will generate both backend and frontend snyk reports
+
+In gitbash:
+
+```shell
+./run-snyk-tests.sh
+```
+
+In powershell:
+
+```shell
+.\run-snyk-tests.sh
+```
+
+These reports will be created and saved in _reports_ folder under _./project-thundercat/_.
+
+---
+
+## Accessibility Tool (pa11y):
+
+This script will generate the accessibility report
+
```

Comment:
Option 1: May want to either add that docker containers need to be up and running for this to work, or see my other comments (option 2)
The existing scripts do not require our containers to be running
-Tests starts the containers one by one and runs the scripts (docker-compose run ....)
-Snyk only needs to read a file and does not need anything running.

Reply:
As discussed offline, I chose the Option 1, since this will be faster to generate these reports without starting and stopping any new/existing containers.

We'll have to generate these reports for all frontend changes, so we want this as quicker as possible.

Updated in commit #https://github.com/code-for-canada/project-thundercat/pull/75/commits/1abb872877dc1b22044be3e812c0a622876157ec